test/e2e/framework/framework.go

```diff
@@ -373,6 +374,11 @@ func (f *Framework) CreateNamespace(baseName string, labels map[string]string) (
 	if ns != nil {
 		f.namespacesToDelete = append(f.namespacesToDelete, ns)
 	}
+
+	if !f.SkipDefaultPSPBinding {
```

Comment:
"Default" is unclear... make the name clearer, like `SkipPermissivePSPBinding`

Reply:
done